OneCommunityGlobal/HighestGoodNetworkApp
# EvianTan - update the readme files

# Description
Due to the dated Readme file, I plan to update it and also create "good" documentation in the near future

## Related PRS (if any):
None

## Main changes explained:
Update the markdown file and upload images

## How to test:
just see the view from this link - https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/tree/Yiyun-readme_updates

## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/5071040/1f948730-3779-428c-bc8a-6e6d199d4a53)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/5071040/992cb2b9-f22e-4d92-8826-11498b3543c6)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/5071040/8db9665d-cf11-4ff6-a059-c320ec4da9ea)
